src/foam/nanos/crunch/Capability.js

```diff
@@ -449,6 +449,7 @@ foam.RELATIONSHIP({
   extends:'foam.nanos.crunch.Renewable',
   sourceModel: 'foam.nanos.auth.User',
   targetModel: 'foam.nanos.crunch.Capability',
+  ids: [ 'id' ],
```

Comment:
I think this is the only important line.

Reply:
yes, that's when relationship allows setting 'ids' to a custom one. The default ids is ['sourceId', 'targetId'].